### ISSUE TITLE
Default REST path should take current app and not /EntityBrowser

### ISSUE THREAD

**Issue by eduard93** [intersystems-ru/iknow-entity-browser#10]
Currently web app goes to `/EntityBrowser/api...`.  It should take current URL and determine correct app automatically.

**Comment by eduard93** [comment]
Is it determined [here](https://github.com/intersystems-ru/iknow-entity-browser/blob/master/src/static/js/source/index.js#L9)?

**Comment by nikitaeverywhere** [comment]
Yep. What is the case? Did you rename `EntityBrowser`?

**Comment by eduard93** [comment]
> What is the case? Did you rename EntityBrowser?

Yes. Well, I added another one. I need EntityBrowser in two namespaces.

**Comment by eduard93** [comment]
I'm planning to replace hardcoded `EntityBrowser` with `window.location.pathname.replace(/\//g, '')`

**Comment by nikitaeverywhere** [comment]
Ah okay, you can simply change this in settings (hopefully. But I think EntityBrowser shares localStorage on the same domain...).
Adding a new feature to iKnowEntityBrowser which will allow to change the namespace would be great.

Note that pathname could also include `/index`, use regex that replaces everything after the second slash.

**Comment by eduard93** [comment]
Okay, I've modified locally to `window.location.pathname.split("/")[1]` and it works except settings page shows incorrect app. But requests go to the correct one.

![image](https://user-images.githubusercontent.com/5127457/43200960-de4e2442-901f-11e8-8f3c-de84f1e8044d.png)

**Comment by nikitaeverywhere** [comment]
It should be `${ getOption("webAppName") || window.location.pathname.split("/")[1] || "EntityBrowser" }`, right?

**Comment by eduard93** [comment]
Removed `getOption("webAppName")` as that way two apps wouldn't work on the same server I think

**Comment by nikitaeverywhere** [comment]
Please keep it - it is used to connect to remote servers. You can set it to `""` for your case to work.

**Comment by nikitaeverywhere** [comment]
Good. I believe you can do PR with `getOption("webAppName") || window.location.pathname.split("/")[1] || "EntityBrowser"`

**Comment by nikitaeverywhere** [comment]
However, this should work from the beginning if you set `""` value [here](https://github.com/intersystems-ru/iknow-entity-browser/blob/ddd0ae56af93183e10695719ff382a862fb8ab2d/src/static/js/settings/values.js#L29). Then, change readme as well. I think adding a correct placeholder to the input would also be great.
